src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int bothside;
```

Comment:
The variable name 'bothside' is misleading in the context of a rectangle, which has 'length' and 'width'. Consider renaming it to 'width' or 'length' to clearly represent the rectangle's properties.